src/asInput/index.jsx

```diff
@@ -14,7 +14,7 @@ export const inputProps = {
   label: PropTypes.oneOfType([PropTypes.string, PropTypes.element]).isRequired,
   name: PropTypes.string.isRequired,
   id: PropTypes.string,
-  value: PropTypes.string,
+  value: PropTypes.oneOfType(PropTypes.string, PropTypes.number),
```

Comment:
Do you need to pass in an array like `PropTypes.oneOfType([PropTypes.string, PropTypes.number])` ?

Reply:
Whoops, yes. I didn't check the JS console on the stories page for this.

I also fixed a console error about 12 text inputs having the same `id` on the "different textual inputs" story, so I defined a separate `id` for each of the components in that story.